src/Symfony/Component/Serializer/Encoder/YamlEncoder.php

```diff
@@ -29,6 +30,10 @@ class YamlEncoder implements EncoderInterface, DecoderInterface
 
     public function __construct(Dumper $dumper = null, Parser $parser = null, array $defaultContext = array())
     {
+        if (!class_exists(Dumper::class)) {
+            throw new RuntimeException('Unable to use YamlEncoder as the Symfony Yaml component is not installed.');
```

Comment:
Maybe we should add the package name here too as suggested in #24563.

Reply:
done